src/doc/en/developer/portability_testing.rst

```diff
@@ -1152,6 +1152,49 @@ are available:
    ptest`` has not been run yet.
 
 
+Testing GitHub Actions locally
+==============================
+
+`act <https://github.com/nektos/act>`_ is a tool, written in Go, and using Docker,
+to run GitHub Actions locally; in particular, it speeds up developing Actions.
+We recommend using `gh extension` facility to install `act`. ::
+
+    [alice@localhost sage]$ gh extension install https://github.com/nektos/gh-act
+
+Extra steps needed for configuration of Docker to run Actions locally can be found on
+`act's GitHub <https://github.com/nektos/act>`_
+
+Here we give a very short sampling of `act`'s capabilities. If you installed standalone
+`act`, it should be called as `act`, not as `gh act`.
+After the set up, one can e.g. list all the available actions::
+
+    [alice@localhost sage]$ gh act -l
+    Stage  Job ID                  Job name                                                   Workflow name                                      Workflow file           Events
+    0      build                   build                                                      Build & Test                                       build.yml               workflow_dispatch,pull_request,push
+    0      test                    Conda                                                      Build & Test using Conda                           ci-conda.yml            push,pull_request,workflow_dispatch
+    0      cygwin-stage-i-b        cygwin-stage-i-b                                           CI cygwin-standard                                 ci-cygwin-standard.yml  push,workflow_dispatch
+    [...]
+
+run a particular action ``foobar`` ::
```

Comment:
I think it would be better to give an example that actually exists (and ideally also include that in the above `gh act -l` output)